Blockzilla/SettingsViewController.swift

```diff
@@ -591,7 +604,7 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
     }
 
     @objc private func toggleSwitched(_ sender: UISwitch) {
-        let toggle = toggles.values.filter { $0.toggle == sender }.first!
+        let toggle = toggles.values.filter {$0.values.filter { $0.toggle == sender } != []}[0].values.filter {$0.toggle == sender }[0]
```

Comment:
nit: whitespacing is not consistent before/after each brace. Can you also explain why this is needing changed? I'm a little confused by it 😅

Reply:
It needs changed because before toggles was a dictionary of blockerToggles. I changed it to a dictionary of dictionaries of blockerToggles. So when we have an index we can find the toggle with toggle[sectionIndex][rowIndex]. Previously, it was necessary to go through all the sections and add the lengths to the row index to find an overall index and then do toggle[overallIndex]. Because of the dictionary of dictionary setup, the filter will just give a dictionary, so a second filter is needed.